lib/order.rb

```diff
@@ -0,0 +1,43 @@
+require './lib/menu.rb'
+require './lib/bill.rb'
+
+class Order
+  attr_reader :order, :current_total, :bill
+  def initialize
+    @order = []
+  end
+
+  def add_to_order(food, quantity)
+    fail "This item is not on the menu" if !item_exist?(food)
+    @order << {item: food, quantity: quantity}
+  end
+
+  def current_total
+    @current_total = calc_total
+  end
+
+  def view_order
+    @order.each do |item|
+      puts "#{item[:item]} x#{item[:quantity]}"
+    end
+    puts current_total
+  end
+
+  private
+
+  def calc_total
+    total = 0
+    @order.each do |item|
+      Menu::MENU.each do |menu_item|
```

Comment:
Good solution to use `Menu` as a namespace for storage and directly reference it!